tests/TestCase/I18n/DateTest.php

```diff
@@ -140,6 +142,12 @@ public function testNice($class)
         $this->assertEquals('Nov 6, 2015', $date->nice());
         $this->assertEquals('Nov 6, 2015', $date->nice(new DateTimeZone('America/New_York')));
         $this->assertEquals('6 nov. 2015', $date->nice(null, 'fr-FR'));
+        
+        $class::setDefaultOutputTimezone('America/Vancouver');
+        $date = new $class('2015-11-06 01:00:00', 'UTC');
+        $this->assertEquals('Nov 5, 2015', $date->nice()); // Why is this failing!?
```

Comment:
Any idea why this is failing? Jumping days by specifying timezones works for times / datetimes but seems not to work for dates?!